src/main/java/com/reviews/demo/service/impl/CommentServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.reviews.demo.service.impl;
+
+import com.reviews.demo.model.Comment;
+import com.reviews.demo.repository.CommentRepository;
+import com.reviews.demo.service.CommentService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CommentServiceImpl implements CommentService {
+    private final CommentRepository commentRepository;
```

Comment:
services and daos should be in the next pr, be more attentive next time about PR content